stompServer.js

```diff
@@ -1,46 +1,59 @@
-var stomp = require('./lib/stomp');
-var StompUtils = require('./lib/stomp-utils');
-var http = require("http");
+var http            = require('http');
 var WebSocketServer = require('ws').Server;
-var EventEmitter = require('events');
-var util = require('util');
+var EventEmitter    = require('events');
+var util            = require('util');
+
+var Stomp           = require('./lib/stomp');
+var StompUtils      = require('./lib/stomp-utils');
+var Bytes           = require('./lib/bytes');
+
+const VERSION = require('./package.json').version;
+
 
 /**
  * STOMP Server configuration
+ *
  * @typedef {object} ServerConfig
  * @param {http.Server} server Http server reference
  * @param {function} [debug=function(args) {}] Debug function
- * @param {string} [path=/stomp] Websocket path
- * */
+ * @param {string} [path=/stomp] WebSocket path
+ * @param {string} [serverName=STOMP-JS/VERSION] Name of STOMP server
+ * @param {number} [heartbeat=[10000, 10000]] Heart-beat; read documentation to config according to your desire
+ */
 
 
 /**
  * @class
  * @augments EventEmitter
- * Create stomp server with config
+ * 
+ * Create Stomp server with config
+ * 
  * @param {ServerConfig} config Configuration for STOMP server
  */
 var StompServer = function (config) {
   EventEmitter.call(this);
-  if (config === undefined) {
+
+  if (config == null) {
     config = {};
   }
+  
   this.conf = {
     server: config.server,
-    serverName: config.serverName || "STOMP-JS/1.1.2",
-    path: config.path || "/stomp",
+    serverName: config.serverName || 'STOMP-JS/' + VERSION,
+    path: config.path || '/stomp',
+    heartbeat: config.heartbeat || [10000, 10000],
     debug: config.debug || function (args) {
+      // console.log(arguments);
     }
   };
-  if (this.conf.server === undefined) {
-    throw "Server is required";
+  
+  if (this.conf.server == null) {
```

Comment:
`!this.conf.server` would trigger an error, when called with a default config

Reply:
I don't understand. Could you elaborate?

Reply:
I was not right about, it will trigger an error. This comparison is not allowed by Bithound, please use ===. To pass it's test!

Reply:
Ok. Thanks.